### DISCUSSION TITLE
free trial

### DISCUSSION THREAD

**Discussion by TanvirHafiz** [abus-aikorea/voice-pro#10] [General]
it is saying 30 mins free trial when i havent even created like 2 mins of TTS, can anyone explain?

**Comment by gauvey** [comment]
Since it is a paid software, please tell me how to purchase it? Where to purchase from?

**Comment by abus-aikorea** [comment]
I apologize for the lack of explanation. Let me provide more details.

All ABUS products are available through purchasing coffee on "Buy Me a Coffee"
Depending on the purchase amount, we will send you a usage voucher for up to 1 month.
The voucher will be provided in the form of a .pyd file via Google Drive. 
This .pyd file has the functionality to remove the 30-minute usage limit for a specified period.
After the usage period expires, the 30-minute usage limit will be restored.

Thank you for your interest in ABUS products. 
If you have any further questions, please feel free to contact us anytime.

![388408704-f1dc1463-0290-4637-9dc2-4df018e0a808](https://github.com/user-attachments/assets/5ced16bd-5c30-49a2-b4cc-6265f09388a1)

**Reply by TanvirHafiz** [reply]
is it a physical purchase? because it should not matter where we are located if its a digital purchase. also is there any lifetime purchase available?

**Reply by TanvirHafiz** [reply]
or its hosted somewhere we can use by puchasing subsctiption  or token like rundiffusion
